app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   def index
     @latest_projects = Project.latest(5)
+    @latest_repositories = Repository.latest(5)
+    @latest_configurations = KalibroConfiguration.latest(5)
```

Comment:
I just noticed it - doesn't this defeat the cache completely? Since it's not inside a cache block it will always be executed before the view is rendered, and always fetch everything.